components/UtilityAnalysisInteraction.tsx

```diff
@@ -57,7 +78,9 @@ const UtilityAnalysisInteraction = ({ GymiProviders, CourseDetails }: { GymiProv
         });
 
         // Sortieren der berechneten Anbieter nach ihrem Gesamt-Score
-        const ratedGymiProvidersList = sortList(mappedProviders);
+        const ratedGymiProvidersList = mappedProviders.sort((a, b) => {
+          return b.totalScore - a.totalScore;
+        });
```

Comment:
the `sortList` function uses too much `any`-types and is somewhat not necessary. Let's use the sort directly here.